src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -50,7 +51,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
-    User user = new User(UUID.randomUUID(), username, Instant.now(), passwordHash);
+    User user = new User(UUID.randomUUID(), username, Instant.now(), passwordHash, about);
```

Comment:
I'm a bit confused about the flow of this program. Is this new User object instantiated when a person signs up for a new account, or when they log in? I'm guessing it's the former since this is `RegisterServlet`?

If it's the former, are we adding a new text input box on the sign-up page so that users can input their about information? Otherwise, won't `about` always be an empty string or null when a person sign up for a new account?

Reply:
Yup, this is how the current logic goes. doPost creates a new user from the text input (where the about is empty). Do you have any suggestions on how to improve it?

Reply:
Maybe we can add a new text input box on the sign-up page for "About information" and label it as optional, so if users input anything, we can store that instead of assigning an empty string to everyone?

Reply:
That sounds like a good idea! I'll make sure to change corresponding registration files.

Reply:
Maybe we should just have the register page focussed entirely on creation of users. You can set the about to a default value here (I would then recommend making a separate constructor/factory function on User that is for registration and nothing more